Models/Topic.model.js

```diff
@@ -0,0 +1,10 @@
+const db = require("../db/index.js");
```

Comment:
This won't connect us to the DB. Have a look for the file that contains the connection. This index file just contains the data